baselines/webworker.generated.d.ts

```diff
@@ -900,6 +900,13 @@ interface WorkerUtils extends Object, WindowBase64 {
     setTimeout(handler: any, timeout?: any, ...args: any[]): number;
 }
 
+interface Canvas2DContextAttributes {
```

Comment:
Yes, add a property `"flavor": "web"` should prevent it from being emitted to worker file